x-pack/test/cases_api_integration/common/lib/api/connectors.ts

```diff
@@ -344,3 +344,26 @@ export const executeConnector = async ({
 
   return mapKeys(res, (_v, k) => camelCase(k)) as ActionTypeExecutorResult<unknown>;
 };
+
+export const executeSystemConnector = async ({
+  supertest,
+  connectorId,
+  req,
+  expectedHttpCode = 200,
+  auth = { user: superUser, space: null },
+}: {
+  supertest: SuperTest.SuperTest<SuperTest.Test>;
+  connectorId: string;
+  req: Record<string, unknown>;
+  expectedHttpCode?: number;
+  auth?: { user: User; space: string | null };
+}): Promise<ActionTypeExecutorResult<unknown>> => {
+  const { body: res } = await supertest
+    .post(`${getSpaceUrlPrefix(auth.space)}/api/cases_fixture/${connectorId}/connectors:execute`)
```

Comment:
I had to create a new route for executing system actions only for testing because the public execute API does not allow the execution of system actions.